### DISCUSSION TITLE
Feature Extraction - new label using formation

### DISCUSSION THREAD

**Discussion by zeelsheladiya** [ravi7522/Football-Prediction#16] [General]
I added some new features using the formation of the player's x and y position and then converted those values to a format like 4-3-3, 4-2-1-3, etc.

Now the issue is it's having around 28 unique values for both teams, so a total of 56 new features which will increase the no of features from the original 44 to 56. 
Can we add those to the model ? 
Will it cause any problems while building the model or not?

**Comment by zeelsheladiya** [comment]
Please provide your opinion on those questions.

**Comment by ravi7522** [comment]
If we add all those values to the model then the issue is it will taking more time will required to train the model. But for sure we can add all those new values to the model to train which might help us to improve the model.

**Reply by zeelsheladiya** [reply]
can we use one hot encoding?

**Reply by ravi7522** [reply]
we can use one hot encoding. But with that we will have 2 issues which are as follows:
1) As per your analysis you have 56 new unique values which might be too much for some model to train as it requires more processing power. Which I don't have.
2) There are chances that the among those all values we will have same values which might cause some problem for model to train. But I am not that sure this will happen or not.

**Comment by ravi7522** [comment]
@zeelsheladiya can we convert those to some specific values as per the position and make some classes.

**Reply by zeelsheladiya** [reply]
I have already labeled it as a team formation format. I guess we need to convert the one hot encoding binary 1d tensor into one numeric value.
